RecoLocalTracker/SiPixelClusterizer/plugins/alpaka/SiPixelRawToCluster.cc

```diff
@@ -152,6 +256,17 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
       fedIds_ = cablingMap_->fedIds();
       cabling_ = cablingMap_->cablingTree();
       LogDebug("map version:") << cablingMap_->version();
+      tTopo_ = &iSetup.getData(trackerTopologyToken_);
+      digiMorphingConfig_.morphingModules.clear();
+      for (const auto& connection : cablingMap_->det2fedMap()) {
+        auto rawId = connection.first;
+        if (rawId == 0)
+          continue;
+        DetId detId(rawId);
+        if (!skipDetId(tTopo_, detId, theBarrelRegions_, theEndcapRegions_)) {
+          digiMorphingConfig_.morphingModules.push_back(rawId);
+        }
+      }
```

Comment:
can this be done only when the topology changes, instead of for every event ?

I guess that topology can only change at run boundaries, not lumisection boundaries ?
Then it could be moved to beginRun (and still done only if the topology changes) ?